wire-runtime/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt

```diff
@@ -22,42 +23,46 @@ data class Person(
     val phone: List<PhoneNumber> = emptyList(),
     val unknownFields: ByteString = ByteString.EMPTY
 ) {
-    object ADAPTER : ProtoAdapter<Person>(FieldEncoding.LENGTH_DELIMITED, Person::class.java) {
-        override fun encodedSize(value: Person): Int = ProtoAdapter.STRING.encodedSizeWithTag(1, value.name) +
-            ProtoAdapter.INT32.encodedSizeWithTag(2, value.id) +
-            ProtoAdapter.STRING.encodedSizeWithTag(3, value.email) +
-            PhoneNumber.ADAPTER.asRepeated().encodedSizeWithTag(4, value.phone) +
-            value.unknownFields.size()
+    companion object {
+        @JvmField
+        val ADAPTER: ProtoAdapter<Person> =
```

Comment:
@Egorand is there a way we can get rid of the explicit type here if we are initializing the property?

Reply:
No, property types are always declared explicitly.